lualib/rspamadm/rescore.lua

```diff
@@ -0,0 +1,300 @@
+local torch = require "torch"
```

Comment:
There is a function called `rspamd_config:has_torch()` to provide some fail safe for those who has no torch somehow:

```lua
if rspamd_config:has_torch() then
  torch = require "torch"
  torch.setnumthreads(1)
end
```

Here is an example of this function. However, it seems that it might be unavailable in `rspamadm` due to lack of `rspamd_config` superglobal. In theory, this check can also be added to the C file.